promscale/cli.md

```diff
@@ -1,9 +1,30 @@
 # Promscale command line interface (CLI) reference
 
-This document gives you information about the configuration flags and arguments supported by the Promscale Connector.
-You can also find information on flags with `promscale_<version> -help`.
-
-**Note:** Flags can be set as environment variables by converting `-` to `_` (if any), and prefixing with `PROMSCALE_`. For example, `db-host` can be set as an environment variable as `PROMSCALE_DB_HOST`.
+This document gives you information about the configuration flags and arguments
+supported by the Promscale Connector. You can also find information on flags
+with `promscale_<version> -help`.
+
+**Note:** Flags can be set as environment variables by converting lowercase to
```

Comment:
Can we add this to a special section i.e. by using `<highlight type="">`

Reply:
done.